feature-libs/product-configurator/rulebased/components/product-title/configurator-product-title.component.html

```diff
@@ -1,7 +1,16 @@
 <ng-container *ngIf="product$ | async as product">
   <div class="cx-general-product-info">
     <div class="cx-title">
-      {{ product.name }}
+      <!-- TODO(issue: #11238): update feature level to the release we are publishing with,
+        It is still 3.1 only because app.module.ts states that we are on 3.1.
+        Finally we must have 3.x, x>=2 here -->
+      <ng-container *cxFeatureLevel="!3.1">
```

Comment:
It was updated to `3.2` in `app.module.ts` in develop branch.

Reply:
Picked

Reply:
We will take care of this once we know exactly what version we will choose for the release. Under this link, you can find the corresponding task: https://github.com/SAP/spartacus/issues/11238 .